src/app/executions/tasks-runtime/tasks-runtime.component.ts

```diff
@@ -36,30 +41,42 @@ export class TasksRuntimeComponent implements OnInit {
     graphModel: TaskProgress[] = [];
     filter: '';
 
-    readonly CREATED_AT_RELATIVE = CREATED_AT_RELATIVE;
+    readonly STARTED_AT_RELATIVE = STARTED_AT_RELATIVE;
     readonly DURATION_SEC = DURATION_SEC;
 
-    private sort = {by: CREATED_AT_RELATIVE, dir: 'asc'};
+    private sort = {by: STARTED_AT_RELATIVE, dir: 'asc'};
 
     ngOnInit(): void {
         this.calculateTimes(this.execution, this.tasks);
     }
 
     private createGraphModel(tasks: TaskExec[], workflowStarted: string) {
         return tasks.map(task => {
+
+            let created_at = task.created_at;
```

Comment:
You can use the new way of [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```javascript
const {created_at, updated_at, started_at, finished_at} = task;
```

Reply:
done